spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkScanBuilder.java

```diff
@@ -108,8 +108,8 @@ public Filter[] pushFilters(Filter[] filters) {
         expr = SparkFilters.convert(filter);
       } catch (IllegalArgumentException e) {
         // converting to Iceberg Expression failed, so this expression cannot be pushed down
-        LOG.warn("Failed to convert filter to Iceberg expression, skipping push down for this expression: {}",
-            filter, e);
+        LOG.info("Failed to convert filter to Iceberg expression, skipping push down for this expression: {} {}",
```

Comment:
I’d maybe put a `.` or `-` in between the two templated values to delineate the two sentences but I’m ok with it as is as well.

Reply:
added a `.` :)